package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hapi-mongodb",
-  "version": "9.0.1",
+  "version": "9.0.2",
```

Comment:
Never bump versions when you submit PRs.

Reply:
version bump reverted